common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -552,8 +553,32 @@ def student_view(self, context):
         self.send_unit_viewed_event()
         # Link current user to children of this block
         self.link_current_user_to_children()
+        #Renders children that are scheduled for review, or returns a message.
```

Comment:
@macornwell Please insert an empty line above this one, and add a space after `#`.